crates/chia-sdk-driver/src/layers/standard_layer.rs

```diff
@@ -0,0 +1,80 @@
+use chia_bls::PublicKey;
+use chia_puzzles::standard::{StandardArgs, StandardSolution, STANDARD_PUZZLE_HASH};
+use chia_sdk_types::Conditions;
+use clvm_traits::{clvm_quote, FromClvm};
+use clvm_utils::CurriedProgram;
+use clvmr::{Allocator, NodePtr};
+
+use crate::{DriverError, Layer, Puzzle, Spend, SpendContext};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct StandardLayer {
+    pub synthetic_key: PublicKey,
+}
+
+impl StandardLayer {
+    pub fn new(synthetic_key: PublicKey) -> Self {
+        Self { synthetic_key }
+    }
+
+    // This lint is ignored since semantically we want spends to consume Conditions.
+    #[allow(clippy::needless_pass_by_value)]
+    pub fn spend(
```

Comment:
do you plan to add hidden_spend now or in the future? :)

Reply:
Probably in the future